launch_testing_ros/test/examples/README.rst

```diff
@@ -0,0 +1,49 @@
+Launch testing examples
```

Comment:
this README is not really discoverable in `launch_testing_ros/test/examples`, can we add a link from a README.md in `launch_testing_ros` to here? Or, also, we could symlink the folder in `launch_testing_ros/examples`, which is easier to discover.

Reply:
I would avoid symlinks in the repository.

Reply:
Ok, maybe just merge this new readme with the one [here](https://github.com/ros2/launch_ros/tree/master/launch_testing_ros#launch_testing_ros).
So the readme is in a discoverable place, and we link to the examples in test/examples from there.

Reply:
Done